coreos-devel/mantle/mantle-9999.ebuild

```diff
@@ -20,6 +20,8 @@ DESCRIPTION="Mantle: Gluing CoreOS together"
 HOMEPAGE="https://github.com/coreos/mantle"
 LICENSE="Apache-2"
 SLOT="0"
+# objcopy/split have trouble with our cross-compiled kolet
```

Comment:
You can define a QA_something var to specifically exclude the arm binary. That way the rest still gets stripped.